src/components/InfoSection/InfoSection.styles.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+export const InfoSectionWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 127px;
```

Comment:
The footer is still a bit off. 2px bottom margin would help

![image](https://user-images.githubusercontent.com/9722792/78070619-66805400-739c-11ea-811f-a3c8a90f5006.png)